api/generate.go

```diff
@@ -45,6 +46,19 @@ func Generate(cfg *config.Config, option ...Option) error {
 		o(cfg, &plugins)
 	}
 
+	for _, p := range plugins {
+		if inj, ok := p.(plugin.GoRootInjector); ok {
+			roots := inj.GoRoots()
+			for _, root := range roots {
+				code.AddGoRoot(code.GoModuleSearchResult{
+					Path:       root.Path,
+					GoModPath:  root.GoModPath,
+					ModuleName: root.ModuleName,
+				})
+			}
+		}
+	}
+
 	for _, p := range plugins {
```

Comment:
```suggestion
		}

```
There's no point in iterating through the plugins twice.